src/components/NcAppContent/NcAppContent.vue

```diff
@@ -159,7 +159,10 @@ import 'splitpanes/dist/splitpanes.css'
 const browserStorage = getBuilder('nextcloud').persist().build()
 const { name: productName } = loadState('theming', 'data', { name: 'Nextcloud' })
 const activeApp = loadState('core', 'active-app', appName)
-const localizedAppName = loadState('core', 'apps', {})[activeApp]?.name ?? appName
+const localizedAppNameState = loadState('core', 'apps', {})
+const localizedAppName = Array.isArray(localizedAppNameState)
+	? ((localizedAppNameState.find(app => app.id === activeApp)?.name) ?? appName)
+	: ((localizedAppNameState[activeApp]?.name) ?? appName)
```

Comment:
Now it's too much for me 😆 Maybe this would work?
```suggestion
const localizedAppName = (Array.isArray(localizedAppNameState)
	? localizedAppNameState.find(app => app.id === activeApp)?.name
	: localizedAppNameState[activeApp]?.name
) ?? appName
```